lib/rucio/common/types/literals/replica_sort.py

```diff
@@ -0,0 +1,18 @@
+# -*- coding: utf-8 -*-
+# Copyright European Organization for Nuclear Research (CERN) since 2012
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from .literal import Literal
+
+ReplicaSortLiteral = Literal['geoip', 'closeness', 'dynamic', 'ranking', 'random']
```

Comment:
Do we want to define `Literal`s like this? Ideally we'd define an `Enum`, but I want to avoid runtime changes and just focus on the annotations for now.